### ISSUE TITLE
Fresh install of base - npm errors

### ISSUE THREAD

**Issue by jerfowler** [themeteorchef/base#181]
After `meteor npm install` we have unmet dependencies:

```
npm WARN eslint-config-airbnb@9.0.1 requires a peer of eslint-plugin-import@^1.7.0 but none was installed.
npm WARN eslint-config-airbnb-base@3.0.1 requires a peer of eslint-plugin-import@^1.7.0 but none was installed
```

Then when I run `meteor` I get this:

```
W20160830-22:20:44.417(-5)? (STDERR) C:\Users\jeremy.fowler\Desktop\Projects\base\.meteor\local\build\programs\server\shell-server
.js:15
W20160830-22:20:44.422(-5)? (STDERR) require("meteor-promise").makeCompatible(Promise, require("fibers"));
W20160830-22:20:44.423(-5)? (STDERR)                           ^
W20160830-22:20:44.424(-5)? (STDERR)
W20160830-22:20:44.425(-5)? (STDERR) TypeError: require(...).makeCompatible is not a function
W20160830-22:20:44.427(-5)? (STDERR)     at Object.<anonymous> (C:\Users\jeremy.fowler\Desktop\Projects\base\.meteor\local\build\p
rograms\server\shell-server.js:15:27)
W20160830-22:20:44.428(-5)? (STDERR)     at Module._compile (module.js:409:26)
W20160830-22:20:44.429(-5)? (STDERR)     at Object.Module._extensions..js (module.js:416:10)
W20160830-22:20:44.431(-5)? (STDERR)     at Module.load (module.js:343:32)
W20160830-22:20:44.443(-5)? (STDERR)     at Function.Module._load (module.js:300:12)
W20160830-22:20:44.445(-5)? (STDERR)     at Module.require (module.js:353:17)
W20160830-22:20:44.446(-5)? (STDERR)     at require (internal/module.js:12:17)
W20160830-22:20:44.447(-5)? (STDERR)     at Object.<anonymous> (C:\Users\jeremy.fowler\Desktop\Projects\base\.meteor\local\build\p
rograms\server\boot.js:101:3)
W20160830-22:20:44.448(-5)? (STDERR)     at Module._compile (module.js:409:26)
W20160830-22:20:44.449(-5)? (STDERR)     at Object.Module._extensions..js (module.js:416:10)
```

Looks like Chimp uses `xolvio-sync-webdriverio` which has an old dependency of meteor-promise@0.4.8 from npm which gets installed, we need a newer 0.7.3 version installed to prevent the error:

```
λ meteor npm list meteor-promise
application-name@1.0.0 C:\Users\jeremy.fowler\Desktop\Projects\base
+-- chimp@0.39.4
| +-- cucumber@1.2.2 (git://github.com/xolvio/cucumber-js.git#ae7ae962abadcfa141c385ec815014c104250de1)
| | `-- meteor-promise@0.7.2
| `-- xolvio-sync-webdriverio@5.1.0
|   `-- meteor-promise@0.4.8
`-- meteor-promise@0.7.3

```

Pull request: #180

**Comment by Digital-Thor** [comment]
I get the same problems on a fresh installation of the MeteorChef Base boilerplate.  I'm using Windows 7 (64 bit).  Meteor version is 1.4.0.1.

**Comment by themeteorchef** [comment]
These have been fixed in 4.8.0 :)
